From 2140aa879c24dcc9dad7e733c77c1bb212fe1ddf Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Gu=C3=B0ni=20M=C3=A1r=20Gilbert?= <gudni.m.g@gmail.com>
Date: Sun, 30 Mar 2025 02:20:24 +0000
Subject: [PATCH] bluez5: disable aics tests
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Temporarily disable aics tests as they can fail
depending on how the tests are executed. Sometimes they pass,
sometimes they fail. The issue has been observed since BlueZ 5.72 to 5.80

Starting with BlueZ 5.80, the tests began failing when using the
ptest-runner script. This is not a new issue in BlueZ 5.80 which is
why the test is disabled with this commit until a solution is found.

See discussion on Github:
https://github.com/bluez/bluez/issues/726
https://github.com/bluez/bluez/issues/683

Upstream-Status: Inappropriate [OE-Specific]

Signed-off-by: Guðni Már Gilbert <gudni.m.g@gmail.com>
---
 unit/test-vcp.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/unit/test-vcp.c b/unit/test-vcp.c
index 6a61ea2..04b92e4 100644
--- a/unit/test-vcp.c
+++ b/unit/test-vcp.c
@@ -2754,7 +2754,7 @@ int main(int argc, char *argv[])
 	tester_init(&argc, &argv);
 
 	test_vocs_unit_testcases();
-	test_aics_unit_testcases();
+	//test_aics_unit_testcases();
 
 	return tester_run();
 }
